src/test/fuzz/addrman.cpp

```diff
@@ -44,7 +44,7 @@ FUZZ_TARGET_INIT(addrman, initialize_addrman)
             addr_man.m_asmap.clear();
         }
     }
-    while (fuzzed_data_provider.ConsumeBool()) {
+    FUZZER_CONTROLLED_LOOP(fuzzed_data_provider) {
```

Comment:
```suggestion
    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool) {
```

Since some loops (prefer to?) use `fuzzed_data_provider.remaining_bytes()`, maybe make this configurable as well? Feel free to ignore the naming suggestion.

Reply:
@apoelstra Are you still working on this? If not, I am happy to pick up

Reply:
Hi, sorry, I can pick this up now.